ledger/controllers/publisher.js

```diff
@@ -484,21 +488,65 @@ v3.identity =
       reply(boom.badData(ex.toString()))
     }
   }
+
+  async function publishersHandleIdentity (request, reply) {
+    const debug = braveHapi.debug(module, request)
+    const {
+      query
+    } = request
+    const stringified = querystring.stringify(query)
+    const baseUrl = process.env.PUBLISHERS_URL
```

Comment:
we should probably read these out of the runtime config for consistency. to do so we will need to copy the `module.exports.publishers` block under eyeshade in `config.js` to under ledger